src/web/app/views/dialogs/Alert.tsx

```diff
@@ -11,18 +11,29 @@ import i18n from 'helpers/i18n';
 import useI18n from 'helpers/useI18n';
 import { AlertProgressContext } from 'app/contexts/AlertProgressContext';
 import { HELP_CENTER_URLS } from 'app/constants/alert-constants';
-import { IAlert } from 'interfaces/IAlert';
+import { IAlert, MessageIcon } from 'interfaces/IAlert';
 
+import {
+  CheckCircleFilled,
+  CloseCircleFilled,
+  ExclamationCircleFilled,
+  InfoCircleFilled,
+} from '@ant-design/icons';
 import styles from './Alert.module.scss';
 
 const renderIcon = (url?: string): JSX.Element => {
   if (!url) return null;
   return <img className={styles.icon} src={url} />;
 };
 
+// color according to antd [design](https://ant.design/components/alert)
```

Comment:
Design team have defined the color system
https://www.figma.com/design/hYEM7EUfjR6NZt9meMzxDp/Beam-Studio-App?node-id=794-3767&node-type=canvas&t=xWzO1BRCkjrsy27w-0